Supplements/Virgo/portalContext-security.xml

```diff
@@ -47,13 +57,24 @@
     <http auto-config="false" access-denied-page="/denied.zul" use-expressions="true"
           entry-point-ref="apromoreAuthenticationEntryPoint" authentication-manager-ref="authenticationManager">
         <intercept-url pattern="/login.zul*" access="permitAll" requires-channel="any"/>
+        <intercept-url pattern="/index.zul*" access="permitAll" requires-channel="any"/>
```

Comment:
It seems incorrect that index.zul can be accessed without ROLE_USER.

Reply:
It is part of the integration with SecurityMicroservice and secured via JWT in MainController.